app/views/newbookings/_form.html.erb

```diff
@@ -2,7 +2,7 @@
 <html lang="en">
 <head>
 </head>
-<%= form_for ([@customer, @customer.newbookings.build]) do |form| %>
+<%= form_for([@newbookings]) do |form| %>
```

Comment:
I don't think this needs to be an array - did you try `form_for(@newbookings)` ?

Reply:
This was a redundant file that I forgot to delete. This code doesn't actually get used anymore.